projects/storefrontlib/src/cms-components/myaccount/order/order-transitional-tokens.ts

```diff
@@ -0,0 +1,7 @@
+import { Order } from '@spartacus/core';
+import { Observable } from 'rxjs';
+
+// TODO: remove this token in 5.0
+export abstract class OrderDetailsServiceTransitionalToken {
+  abstract getOrderDetails(): Observable<Order>;
+}
```

Comment:
This is a picky comment about naming things:  Since this is an abstract class and not a token, should we call this something else than "token?"

`OrderDetailsTransitionalService` comes to mind.

Unless we have other dependencies using this pattern already...

Reply:
Indeed I just saw the existing pattern in `user-transitional-tokens.ts`.  We leave it as is, it's better do be consistent with the naming.